### PR TITLE
move intro to iac with yaml workshop to on demand videos

### PR DESCRIPTION
**Pull request by sean1588**
Fixes: https://github.com/pulumi/marketing/issues/367

@shughes26  and I converted this workshop [page](https://www.pulumi.com/resources/introduction-to-infrastructure-as-code-with-yaml/) to a video on demand page. I believe this is ready, except for the video link which is pending. We could not find the video to link here on the pulumi tv youtube channel so we added one in here as a placeholder temporarily. Will update this as soon as we get the video.

### CODE REVIEW

**Review by zchase — APPROVED** [review]
LGTM! Thank you!!

File: themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md [hunk 1]
```diff
@@ -3,13 +3,11 @@
 title: "Introduction to Infrastructure as Code with Pulumi YAML"
```
Comment (SaraDPH) [inline themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md:3]: title: Workshop: Introduction to Infrastructure as Code with Pulumi YAML

Comment (sean1588) [reply themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md:3]: @shughes26 can you get this title updated to be what Sara is describing above?

Comment (shughes26) [reply themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md:3]: https://github.com/pulumi/pulumi-hugo/pull/1739 created a new PR with the change

Comment (shughes26) [reply themes/default/content/resources/introduction-to-infrastructure-as-code-with-yaml/index.md:3]: Looks like the updated title exceeds the title character limit by 5 characters (60 characters max)

**Pull request merged**